packages/slate-theme/src/templates/customers/order.liquid

```diff
@@ -35,10 +35,23 @@
           {% if line_item.fulfillment %}
             <div>
               {%- assign created_at = line_item.fulfillment.created_at | date: format: 'month_day_year' -%}
-              {{ 'customer.order.fulfilled_at' | t: date: created_at }}
-              {% if line_item.fulfillment.tracking_number %}
-                <a href="{{ line_item.fulfillment.tracking_url }}">{{ line_item.fulfillment.tracking_company }} #{{ line_item.fulfillment.tracking_number}}</a>
-              {% endif %}
+                <div>
```

Comment:
All the `div` elements look weird, for sure.

I tried multiple `p` elements, but then the area got really huge from the margins the inherit browser style added.  I thought it looked too weird, but maybe it's fine for a demo.   I also tried putting all the content in one `p` tag sounded really strange in a screen reader - no natural pauses for the different pieces of information.

The `div` was used to have a block element that didn't use a lot of space.  

I also had a `dl` implementation which I was pretty happy with:

```
<dl>
  <dt>Company</dt>
  <dd>FedEx</dd>
  <dt>Tracking number<dt>
  <dd>12345678</dd>
</dl>
```

Which was a fair use of the description list according to a couple sources I checked.  Example: [Mozilla web doc - dl](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl).  I abandoned it, however, when I got to the tracking URL though - as this looked too weird:

```
  <dt>Tracking URL<dt>
  <dd>http://www.fedex.com/some-params/that-could/mean-anything/tracking?q=78714763445730&foo=bar-for-days</dd>
```

okay, I'll think of something else because all these `div` elements are bugging me.

Reply:
Went with description list:

![](http://take.ms/gUvZY)

Reply:
I noticed the "Company" translation isn't scoped like "**Tracking** link" and "**Tracking** number" , so should it be more explicit as "**Shipment** Company" or "Carrier" like it is in the admin: http://take.ms/DiZKi ?

Reply:
"Carrier" works for me @paulmason 😃 

Care to make a PR to get it changed?

Reply:
No worries, just added it here: https://github.com/Shopify/slate/pull/319 @NathanPJF